cluster/addons/calico-policy-controller/cni-network-configmap.yaml

```diff
@@ -0,0 +1,23 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
```

Comment:
Yep, spotted that but haven't pushed it yet.

Still having some trouble getting the addon-manager to actually create the ConfigMap though.  @mikedanese any thoughts on how make this play nice with the addon manager?